enterprise/frontend/src/embedding-sdk/cli/utils/get-package-version.ts

```diff
@@ -11,28 +11,57 @@ export const hasPackageJson = async () => {
   }
 };
 
-export const getPackageVersion = async (packageName: string) => {
+type DependencyMap = Record<string, string>;
+
+export const getPackageVersions = async (...packageNames: string[]) => {
+  const versions: Record<string, string> = {};
+
+  const projectDeps = await getProjectDependenciesFromPackageJson();
```

Comment:
This handles the edge case for when the project is not yet `yarn install`'d, like what Jeff ran into earlier.